pkg/console/subresource/deployment/deployment.go

```diff
@@ -259,3 +259,9 @@ func IsReady(deployment *appsv1.Deployment) bool {
 	logrus.Printf("Deployment is avalable: %v \n", deployment.Status.ReadyReplicas >= 1)
 	return deployment.Status.ReadyReplicas >= 1
 }
+
+func IsReadyAndUpdated(deployment *appsv1.Deployment) bool {
```

Comment:
Should this be `IsAvailableAndUpdated()`?  Or `IsExpectedGeneration()`?  Naming is hard, I dunno.

`if IsUpdating() { `  vs `if !IsUpdating() {`?

Reply:
+1 for `IsAvailableAndUpdated()`